toopher/__init__.py

```diff
@@ -110,106 +122,77 @@ def validate(self, data, request_token=None, ttl=DEFAULT_IFRAME_TTL):
 
         return data
 
-    def signature(self, data):
+    def _signature(self, data):
         to_sign = urllib.urlencode(sorted(data.items())).encode('utf-8')
         secret = self.client.client_secret.encode('utf-8')
         return base64.b64encode(hmac.new(secret, to_sign, hashlib.sha1).digest())
 
-    def get_oauth_uri(self, uri, params, ttl):
+    def _get_oauth_signed_url(self, uri, params, ttl):
```

Comment:
Shouldn't the signature be `_get_oauth_signed_url(self, url, ttl, **kwargs)` to match the style of others?